_posts/2020-08-18-scaffolding-static-site-with-cdk.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: post
+title: Scaffolding your AWS static site insfrastructure with CDK
+date: 2020-08-24
+tags: aws cdk static-site react angular cloud-development-kit infrastructure-as-code
```

Comment:
angular?

Reply:
as much as I hate it, I think its better for SEO this way

Reply:
may as well add Vue in then too 😂